docs/references/javascript/sign-up/sign-up.mdx

```diff
@@ -49,9 +49,16 @@ Also, the attributes of the `SignUp` object can basically be grouped into three
 
 This method initiates a new sign-up flow. It creates a new `SignUp` object and de-activates any existing `SignUp` that the client might already had in progress.
 
-The form of the given `params` depends on the configuration of the instance. Choices on the instance settings affect which options are available to use.
+What you pass to `params` depends on the [configuration](/docs/authentication/configuration/sign-up-sign-in-options) of your application instance.

```

Comment:
```suggestion
What you pass to `params` depends on which [sign-in options](/docs/authentication/configuration/sign-up-sign-in-options) you have enabled in your Clerk application instance.
```
Just making it more explicit so they don't have to click through to understand what we mean.